.flake8

```
@@ -10,11 +14,7 @@ ignore=
     D301,D302,
     # Docstring Content
     D400,D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417
-
+    # Type Annotations
+    TYP002,TYP003,TYP101,TYP102,TYP204,TYP206
 per-file-ignores =
-    **/tests/**:D101,D102,D103,D105,D106,S106
-
-application_import_names=pydis_site
-exclude=__pycache__, venv, .venv, **/migrations
-import-order-style=pycharm
-max-line-length=100
+    **/tests/**:D101,D102,D103,D105,D106,S106,TYP
```

Is there a reason to specify each one?
```suggestion
    **/tests/**:D1,S106,TYP
```

---

It’s not each one

---

Does it not overlap with with the normal ignores? I thought it did and in that case it would indeed be all the ones that start with `D1`.

---

You’re right, sorry, I missed that this was the per-file line